.github/workflows/2x-kubernetes-version-test.yaml

```diff
@@ -0,0 +1,128 @@
+name: 2.x Kubernetes Version Tests
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'next'
+
+jobs:
+  test-kubernetes-1-19-dbless:
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16'
+    - name: cache go modules
+      uses: actions/cache@v2.1.6
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+    - name: checkout repository
+      uses: actions/checkout@v2
+    - name: Kubernetes v1.19.x DBLESS Integration Tests
+      run: KONG_CLUSTER_VERSION=v1.19.11 make test.integration.dbless
+      working-directory: ./railgun
+
+  test-kubernetes-1-20-dbless:
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16'
```

Comment:
We can retrieve version using runtime.Version(), or corresponding env environment variables.

Reply:
I'm not sure what the relevance is here: this option tells the `setup-go` action which versions of Go we accept for builds (e.g. latest `1.16.x` ultimately).

Reply:
shouldn't you just read go-lang version from the environment instead of hardcode ?

Reply:
@ccfishk, this line sets up Go for the build process. Before this, there is no Go compiler installed whatsoever. Here we specify what version of Go will be available on the worker.

Reply:
Please `unresolve` if there's any follow up here, but as Michal stated: there's no Golang available _before_ this step and this line simply informs which Golang to install.

Reply:
@shaneutt  understand this is the first line we set the expected golang version. It is good. 👍🏻

Reply:
@mflendrich thanks for pointing this out.